internal/provisioner/external_provisioner.go

```diff
@@ -123,6 +125,24 @@ func (provisioner *ExternalProvisioner) ProvisionCluster(cluster *model.Cluster)
 		if err != nil {
 			return errors.Wrap(err, "failed to deploy pgbouncer manifests")
 		}
+
+		vpc := cluster.ProvisionerMetadataExternal.VPC
+		// Update PgBouncer ConfigMap
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cancel()
+		logger.Info("Updating PgBouncer ConfigMap")
+
+		sqlStore, ok := provisioner.clusterUpdateStore.(*store.SQLStore)
+		if !ok {
+			logger.Warn("Store is not a SQLStore, skipping PgBouncer ConfigMap update")
+			return nil
+		}
```

Comment:
Let's not do this. Instead we should extend the store interface that we have here to have any missing calls needed to pass in later to `UpdatePGBouncerConfigMap`. Let me know if you have any questions on how to do that.